content/en/docs/ops/ambient/getting-started/test.sh

```diff
@@ -65,7 +65,7 @@ _verify_contains snip_verify_traffic_notsleep_to_productpage "$snip_verify_traff
 _verify_contains snip_adding_your_application_to_the_ambient_mesh_1 "$snip_adding_your_application_to_the_ambient_mesh_1_out"
 
 # test traffic after ambient mode is enabled
-_verify_contains snip_adding_your_application_to_the_ambient_mesh_3 "$snip_adding_your_application_to_the_ambient_mesh_3_out"
+_verify_contains snip_adding_your_application_to_the_ambient_mesh_3 "received netns, starting proxy"
```

Comment:
@frankbu I decided to do the validation directly on the test script as the snips keeps on failing. I see someone did the same before here on line 77

Reply:
test failed again... running out of ideas here...

Reply:
```VERIFY FAILED snip_adding_your_application_to_the_ambient_mesh_3 (timeout after 120s):
received:
""
expected:
"received netns, starting proxy"```

Reply:
Might be worth troubleshooting by dropping the grep in that snip and just having it dump the logs, just to see what is being logged in the ztunnel pods.

Reply:
thanks @dhawton I found that one, I tried several combinations to get that snippet of the doc in but the test keeps on failing for different reasons. The idea here is to show the logs after the ns labeling, but when I use the whole output like:
```
kubectl logs ds/ztunnel -n istio-system  | grep inpod
Found 3 pods, using pod/ztunnel-jrxln
inpod_enabled: true
inpod_uds: /var/run/ztunnel/ztunnel.sock
inpod_port_reuse: true
inpod_mark: 1337
2024-03-26T00:02:06.161802Z  INFO ztunnel::inpod::workloadmanager: handling new stream
2024-03-26T00:02:06.162099Z  INFO ztunnel::inpod::statemanager: pod received snapshot sent
2024-03-26T00:41:05.518194Z  INFO ztunnel::inpod::statemanager: pod WorkloadUid("7ef61e18-725a-4726-84fa-05fc2a440879") received netns, starting proxy
```

the test `_verify_like` or `_verify_contains` doesn't match but I can see a similar print on the artifacts output, my last attempt just removed the snips.sh out reference and match the contains directly like:
```
_verify_contains snip_adding_your_application_to_the_ambient_mesh_3 "received netns, starting proxy"
```
but as you can see the output is empty

Reply:
that's the thing, on my previous validations I was able to see an output like described above but one pod instead of 3 and obviously different time stamps and workloadid but neither the `_verify_like` or `_verify_contains` matched anything...

Reply:
I can't see the output of the log really, I can only tell grep had no match... the received netns statement is ~2 months old, so it's in the commit of Istio that the docs is using atm. Wonder if it's not starting up, or hasn't finished initializing when the test is being run?